README.md

```diff
@@ -131,6 +131,11 @@ To the extent possible under law, all contributors have waived all copyright and
 
 - [Dados abertos do Rio Grande do Sul](http://dados.rs.gov.br) &mdash; Dados de orgãos púplicos estaduais.
 - [Dados Prefeitura de Porto Alegre](http://datapoa.com.br) &mdash; Dados de orgãos e entidades de administração pública municipal.
+- [Dados abertos TCE-RS](http://dados.tce.rs.gov.br/) &mdash; Dados abertos do tribunal de contas do estado.
```

Comment:
Please remove the trailing `/` in the URL

Reply:
Solved